docs/08_jenkins_testing_enviroment.md

```diff
@@ -0,0 +1,126 @@
+# Jenkins testing environment deployment
+
+> Summary of the proposed change
+
+Deploy Jenkins testing environment to be able to integrate it with the Metrics app.
+
+# References
+
+> Link to supporting documentation, GitHub tickets, etc.
+
+[Jenkins](https://jenkins.io/)  
+[Google Cloud Jenkins application](https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/jenkins/README.md)
+[Amazon AWS](https://aws.amazon.com/)
+
+# Motivation
+
+> What problem is this project solving?
+
+Creating the Jenkins test environment will provide the data for the Metrics application.
+The Jenkins test environment will give us the ability to test the whole process of importing and displaying the project metrics data.
+
+# Goals
+
+> Identify success metrics and measurable goals.
+
+* Jenkins server is running and available for testing by anyone in the team.
+* Configuration of the Jenkins to run the tests from the test project. 
+
+# Non-Goals
+
+> Identify what's not in scope.
+
+The creation of the test project is not in the scope of this project.
+
+# Design
+
+> Explain and diagram the technical design
+
+`Amazon AWS instance` -> `Jenkins Server`
+
+> Identify risks and edge cases
+
+Running the builds in the separate docker containers could require an additional configuration.
```

Comment:
Could you please clarify this point a bit more? Not sure where do we get separate docker containers? Is it from AWS deployment instructions?